debian/build.py

```diff
@@ -40,20 +41,18 @@ def __str__(self):
         self.pkg, self.arch, self.distro, self.version, self.revision)
 
 def main():
-  architectures=('amd64', 'arm', 'arm64')
+  architectures=('amd64',)
   distros=('xenial',)
 
   packages = {
     'kubectl': (
-      ('1.3.7', '00'),
```

Comment:
Yes, indeed. I might as well go and move this into a simple JSON config file, cause I need a way to test different builds.